### ISSUE TITLE
Adding\delete files to the .torrent file

### ISSUE THREAD

**Issue by master255** [arvidn/libtorrent#5015]
@arvidn 
I have a need to periodically update the contents of the .torrent file. However, users have already downloaded the old .torrent file and so that they do not download all the files again, it would be great to give them a new torrent file to update.

To download (and use) a file from an .torrent file, users use the .torrent file hash and file number.

The problem is that the hashes stored in the .torrent file are calculated for pieces and cannot be calculated for each file.
So that users don't lose old files, they need to use a hash of the file they are using, not a general hash of the .torrent file (and his number). I can't calculate that hash right now.

In other words, it is necessary to add thousands of .torrent files to one .torrent file.
This will create one more hash for each file, but will allow you to update the data without losing the old data.

**Comment by arvidn** [comment]
libtorrent supports this: https://www.bittorrent.org/beps/bep_0038.html
just make sure to pad your files in the .torrent, to align the pieces.

**Comment by master255** [comment]
@arvidn Thank you! The library is very cool.
I don't see your payment for [my torrent player](https://play.google.com/store/apps/details?id=com.media.library). You should buy it, then I'll come to Stockholm and buy you a coffee :-)
